ui/apps/platform/src/Containers/ComplianceEnhanced/Coverage/CheckDetailsTable.tsx

```diff
@@ -0,0 +1,93 @@
+import { getClusterResultsStatusObject } from './compliance.coverage.utils';
+
+export type CheckDetailsTableProps = {
+    currentDatetime: Date;
+    profileName: string;
+    tableState: TableUIState<ClusterCheckStatus>;
+};
+
+function CheckDetailsTable({ currentDatetime, profileName, tableState }: CheckDetailsTableProps) {
+    return (
+        <>
+            <Table>
+                <Thead>
+                    <Tr>
+                        <Th>Cluster</Th>
+                        <Th>Last scanned</Th>
+                        <Th>Compliance status</Th>
+                    </Tr>
+                </Thead>
+                <TbodyUnified
+                    tableState={tableState}
+                    colSpan={3}
+                    errorProps={{
+                        title: 'There was an error loading results for this check',
+                    }}
+                    emptyProps={{
+                        message: 'No results found for this check',
+                    }}
+                    filteredEmptyProps={{
+                        title: 'No results found',
+                        message: 'Clear all filters and try again',
+                    }}
+                    renderer={({ data }) => (
+                        <Tbody>
+                            {data.map((clusterInfo) => {
+                                const {
+                                    cluster: { clusterId, clusterName },
+                                    lastScanTime,
+                                    status,
+                                } = clusterInfo;
+                                const clusterStatusObject = getClusterResultsStatusObject(status);
+                                const firstDiscoveredAsPhrase = getDistanceStrictAsPhrase(
+                                    lastScanTime,
+                                    currentDatetime
+                                );
+
+                                return (
+                                    <Tr key={clusterId}>
+                                        <Td dataLabel="Cluster">
+                                            <Link
+                                                to={generatePath(coverageClusterDetailsPath, {
+                                                    clusterId,
+                                                    profileName,
+                                                })}
+                                            >
+                                                {clusterName}
+                                            </Link>
+                                        </Td>
+                                        <Td dataLabel="Last scanned">{firstDiscoveredAsPhrase}</Td>
+                                        <Td dataLabel="Compliance status">
+                                            <Tooltip content={clusterStatusObject.tooltipText}>
+                                                <Button isInline variant={ButtonVariant.link}>
```

Comment:
Observing and asking, not necessarily suggesting change:
* A modal is in component state in all cases that come to mind.
* Does that seem reasonable from user viewpoint? For example, not possible to share a link to any state not in page address query string.

Reply:
Good thought. You're basically asking if we should consider providing a way to include modal state in the url? That seems like a worthwhile discussion to me.